envs/conda_build_config.yaml

```diff
@@ -333,9 +333,9 @@ cuda_levels:
   - 3.7;5.2;6.0;7.0;7.5     # [x86_64]
   - 3.7;6.0;7.0;7.5         # [ppc64le]
 cuda_levels_details:
-  - sm_37,sm_52,sm_60,sm_61,sm_70,compute_75  # [x86_64]
-  - sm_37,sm_60,sm_70,compute_75              # [ppc64le]
+  - sm_37,sm_52,sm_60,sm_61,sm_70,sm_75,compute_75        # [x86_64 and cudatoolkit == '10.2']
```

Comment:
We should probably merge `cuda11_levels` with `cuda_levels` as well. However, you might need to leave in `cuda11_levels_details` for now so that it doesn't break TF. I think you will need to merge in the new cuda_levels_details, then fix TF (and anything else that use `cuda11_levels_details`), then remove `cuda11_levels_details`.

Reply:
Yes, I have been thinking so too regarding the `cuda_levels`. Will add that change.

Reply:
I have added a PR for TF - https://github.com/open-ce/tensorflow-feedstock/pull/65 to fix this.

Reply:
yeah, this is a better idea.

Reply:
If we merge TF PR, along with this PR, I think we would not break TF build.